src/app/services/payment/international-payment.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { InternationalPayment } from 'src/app/models';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class InternationalPaymentService {
+  constructor(private http: HttpClient) { }
+
+  createInternationalPayment(payment: InternationalPayment): Observable<any> {
+    return this.http.post<InternationalPayment>(
+      `${environment.api_url}/transactions/card-foreign/`,
+      payment
+    );
+  }
+}
```

Comment:
@patrickf949 This should be in the existing `payment-service` file. Just add **createInternationalPayment** method to the **PaymentService** class.

Reply:
thanks @Victoradukwu Done